pacmd/__init__.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+"""Sets default audio sink
+
+ Synopsis: <trigger> <query>"""
+
+import subprocess
+from albert import *
+
+__title__ = "Pulseaudio cmd"
+__version__ = "0.1.0"
+__triggers__ = "pacmd "
+__authors__ = "c4pQ"
+__exec_deps__ = ['pacmd']
+
+iconPath = iconLookup('audio-headphones')
```

Comment:
see https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/blob/master/pulseaudio-text.svg i dunno if we can expect that 'pulseaudio-text' is on every system but id rather use this one